core/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@uiw/react-md-editor",
-  "version": "3.23.6",
+  "version": "3.24.0",
```

Comment:
@RARgames Do not modify the version number information. This will trigger CI to automatically publish to npm. There is a separate `script` for the modified version.

https://github.com/uiwjs/react-md-editor/blob/bcced73b7bd35bf26281f7cdfad6b41eeb5e2d9a/package.json#L13

The scripts will have the same modified version, but the versions are inconsistent, which will cause the installation example to not be the local version.